src/helpers.js

```diff
@@ -100,6 +100,10 @@ export function clearCache() {
 export async function queryWebsoc(params) {
     // Construct a request to PeterPortal with the params as a query string
     const url = new URL(PETERPORTAL_WEBSOC_ENDPOINT);
+    // TODO: API may return empty result if params has empty sectionCodes
```

Comment:
Can you investigate this further? See if the API docs mention anything about this, and if not try looking through the PeterPortal API source code for the keyword `sectionCodes` to find where it handles that parameter. If that seems too hard we can discuss more offline about how to dive deep into stuff like this.

I'm asking because knowing more about this will help make sure this code is bug-free.

Reply:
Got it

Reply:
Could you try to reproduce the GE-III bug? I tried using multiple devices, but the correct results showed up.